platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpJsonHelper.java

```diff
@@ -427,8 +427,13 @@ public String requestString(int timeout,
             conn.setReadTimeout(timeout > 0 ? timeout : 60000);
             try {
                 conn.setRequestMethod(method);
+                //drop a hint for server side that we want to receive application/json
+                conn.addRequestProperty(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON);
```

Comment:
`HttpJsonHelper` is __JSON__ helper but still it contains `requestString` method, don't you think that this change may affect code which uses `requestString` method ?